e2e/test/prerequisites/iot_config.cmd_template

```diff
@@ -14,6 +14,13 @@ set IOTHUB_CONN_STRING_CSHARP=
 set IOTHUB_DEVICE_CONN_STRING=
 set IOTHUB_DEVICE_CONN_STRING2=
 
+@REM IoT Hub Module connection string:
+set IOTHUB_MODULE_CONN_STRING=
+set IOTHUB_MODULE_CONN_STRING2=
+
+@REM EventHub consumer group:
+set IOTHUB_EVENTHUB_CONSUMER_GROUP=e2e-tests
```

Comment:
This is incorrect - it's already added below. Also, please don't add values in this file - we'd like it generic.